index.js

```diff
@@ -8,7 +8,27 @@ const token = envy().token;
 let playerList = [];
 let captainList = [];
 
-const prefix = '!'
+const prefix = '!';
+
+//abstracting addPlayer will let us use it in other scenarios, esp. automated testing.
+function addPlayer(player, msg = undefined) {
+    if (playerList.includes(player)) {
+        if (msg != undefined) { msg.reply(`You are already queued up! If you would like to leave, please use ${prefix}leave`); };
+        return [...playerList];
+    };
+    return [...playerList, player];
+};
+
+//abstracting join lets us edit and see it easier, instead of inside the tangle of ifs in the callback. 
+function join(msg) {
+    msg.delete();
+    if (playerList.length > (captainList.length * 2)) {
```

Comment:
I think this should be >=, as we don't want a player do be added to the list if f.e. the captain list length is 5 and the player list length is 10. In this case it would still add the 11th player.

Reply:
You right, haha, woops.